sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -278,8 +278,10 @@ Licensed under the MIT License.
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>3.1.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
         <configuration>
-          <failsOnError>false</failsOnError>
-          <failOnViolation>false</failOnViolation>
+          <suppressionsLocation>../../../eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml
```

Comment:
Can you please remove the above comment regarding CosmosSkip. Since this is now complete :)